### ISSUE TITLE
Aborted sessions don't kill requests to grid

### ISSUE THREAD

**Issue by VitaliLiavitski** [zebrunner/mcloud-grid#5]
If we aborte Jenkins job, requests have status in progress for using phones and simulators

**Comment by vdelendik** [comment]
it might be not a bug but let's investigate if we can handle somehow abort use-case on hub layer. By default aborted sessions should be disconnected by timeout. 
Pay attention to the adb connection which might occupy device for longer time than timeout

**Comment by vdelendik** [comment]
fixed by #54.
testing scenario should be:
1. start iOS test or job via single device (ask @vdelendik to prepare explicit branch with rather slow steps)
2. kill iOS jobs in the middle and start timer to track delay
-> iOS device is occupied either in mcloud/grid/console and in stf
3. start again iOS job vs the same device
-> job is started, request added to queue as device still occupied in grid
4. tail appium device log to detect:
```
2022-01-24 23:02:25:519 - [HTTP] --> DELETE /wd/hub/session/60cd4cef-c063-4ecb-8beb-bf422598784c
2022-01-24 23:02:25:519 - [HTTP] {}
2022-01-24 23:02:25:521 - [debug] [W3C (60cd4cef)] Encountered internal error running command: NoSuchDriverError: A sessio                                                                                                                   n is either terminated or not started
```
-> as only previous session removed in appium (1-4 min) new iOS job should occupy device and start testing


So goal to identify that pause in occupation is upto 4 minutes instead of 60m as in v1.0

**Comment by vdelendik** [comment]
tested on my own
